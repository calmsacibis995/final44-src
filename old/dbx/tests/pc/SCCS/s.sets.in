h17623
s 00013/00000/00000
d D 5.1 90/06/25 15:37:31 bostic 1 0
c date and time created 90/06/25 15:37:31 by bostic
e
u
U
t
T
I 1
whatis Color
whatis ColorSet
stop in p
run
step
where
step
print sets.s
step
step
step
where
print sets.s
E 1
