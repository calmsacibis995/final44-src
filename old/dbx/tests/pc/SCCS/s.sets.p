h26888
s 00018/00000/00000
d D 5.1 90/06/25 15:37:34 bostic 1 0
c date and time created 90/06/25 15:37:34 by bostic
e
u
U
t
T
I 1
program sets (input, output);
type
    Color = (RED, BLUE, GREEN);
    ColorSet = set of Color;
var
    s : ColorSet;

procedure p (var s : ColorSet);
begin
    s := [RED, BLUE];
end;

begin
    p(s);
    if BLUE in s then begin
	s := s - [BLUE];
    end;
end.
E 1
