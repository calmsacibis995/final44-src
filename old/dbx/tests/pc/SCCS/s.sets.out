h53278
s 00024/00000/00000
d D 5.1 90/06/25 15:37:32 bostic 1 0
c date and time created 90/06/25 15:37:32 by bostic
e
u
U
t
T
I 1
reading symbolic information ...
type Color = (RED, BLUE, GREEN);
type ColorSet = set of Color;
[1] stop in p
[1] stopped in p at line 9
    9   begin
stopped in p at line 10
   10       s := [RED, BLUE];
p(s = []), line 10 in "sets.p"
program(), line 14 in "sets.p"
main(0x1, 0x7fffe7e8, 0x7fffe7f0) at 0xa9
stopped in p at line 11
   11   end;
[RED, BLUE] 
stopped in program at line 15
   15       if BLUE in s then begin
stopped in program at line 16
   16   	s := s - [BLUE];
stopped in program at line 18
   18   end.
program(), line 18 in "sets.p"
main(0x1, 0x7fffe7e8, 0x7fffe7f0) at 0xa9
[RED] 

E 1
